seed/challenges/06-information-security-and-quality-assurance/advanced-express-tools.json

```diff
@@ -168,8 +168,8 @@
       "id": "5895f70cf9fc0f352b528e67",
       "title": "Implement the Serialization of a Passport User",
       "description": [
-        "Right now we're not loading an actually users object since we havn't set up our database. This can be done many different ways, but for our project we will connect to the database once when we start the server and keep a persistant connection for the full life-cycle of the app.",
-        "To do this, add MongoDB as a depenacy and require it in your server. (<code>const mongo = require('mongodb').MongoClient;</code>)",
+        "Right now we're not loading an actually users object since we haven't set up our database. This can be done many different ways, but for our project we will connect to the database once when we start the server and keep a persistent connection for the full life-cycle of the app.",
+        "To do this, add MongoDB as a dependancy and require it in your server. (<code>const mongo = require('mongodb').MongoClient;</code>)",
```

Comment:
There are several "dependancy" in the json, I looked in the web and it seems to be a valid word, that's why I didn't changed it. But can fix them, no problem.